addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -213,6 +213,8 @@ export class Many2XAutocomplete extends Component {
     }
 
     async loadOptionsSource(request) {
+        // TODO if request is empty string, call search directly to avoid useless
```

Comment:
nope, the return of name_search is the name_get on the results. Besides, python side name_search does what you say

Reply:
Indeed, stupid remark :D